### DISCUSSION TITLE
Add Variables with Nuclei SDK

### DISCUSSION THREAD

**Discussion by alban-stourbe-wmx** [projectdiscovery/nuclei#5317] [Q&A]
Is there a way to add variables (-V option) via the Nuclei SDK? Or does it make a PR to add a function like nuclei.WithVariables?
Or  we have to use types.Options 

Thanks in advance and have a nice day!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
